### ISSUE TITLE
Model Optimizer Support for Darknet

### ISSUE THREAD

**Issue by mmaaz60** [openvinotoolkit/openvino#361]
When is the next expected release date of OpenVino toolkit? Any plans to add direct support of models trained using darknet framework as converting to Tensorflow does sacrifice some accuracy.

Thanks

**Comment by jgespino** [comment]
Hi @mmaaz60 

We recently released OpenVINO toolkit 2020.1, check out the [release notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes) and you can download [here](https://software.intel.com/en-us/openvino-toolkit/choose-download).

Could you share the commands you are using to convert from DarkNet to TensorFlow and TensorFlow to IR format? Do you see the accuracy drop with the latest release?

Could you provide your weights file for us to reproduce? 

Regards,
Jesus

**Comment by lazarevevgeny** [comment]
There are no plans to support DarkNet directly.

**Comment by lazarevevgeny** [comment]
Closing as not an issue but rather than a quesion.
